packages/cli-repl/package.json

```diff
@@ -69,5 +69,8 @@
     "@types/pino": "^6.3.3",
     "@types/read": "^0.0.28",
     "@types/text-table": "^0.2.1"
+  },
+  "optionalDependencies": {
+    "mongodb-client-encryption": "^1.1.0"
```

Comment:
I think it probably should be in service-provider-server so that we can test it fully there and within the shell-api package, but if it's any additional complexity we can keep it here for now

Reply:
@aherlihy Done, moved it and added a dummy test in the shell-api package :slightly_smiling_face: